.github/workflows/extra.yml

```diff
@@ -101,6 +101,7 @@ jobs:
           -var "source_image_name=${{ fromJSON(env.FAT_IMAGES)['cluster_image'][matrix.build.source_image_name_key] }}" \
           -var "image_name=${{ matrix.build.image_name }}" \
           -var "inventory_groups=${{ matrix.build.inventory_groups }}" \
+          -var "use_blockstorage_volume=true" \
```

Comment:
So this will be for all clouds. Is that OK? If not, I would set it in e.g. `environments/.stackhpc/LEAFCLOUD.pkrvars.hcl` instead.